kyuubi-server/src/test/scala/org/apache/kyuubi/operation/PlanOnlyOperationSuite.scala

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
`PlanOnlyOperationSuite.scala` to `KyuubiPlanOnlyOperationSuite.scala` will be better.

Reply:
?